2012-10-23 20:02:09 Error: [CAS_AuthenticationException] 
#0 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS/Client.php(1221): CAS_Client->validateCAS20('https://login.u...', '', NULL)
#1 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS/Client.php(1080): CAS_Client->isAuthenticated()
#2 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS.php(1100): CAS_Client->forceAuthentication()
#3 /Users/raj/Sites/webspace/sso/app/Controller/Component/CasAuthComponent.php(40): phpCAS::forceAuthentication()
#4 [internal function]: CasAuthComponent->startup(Object(UsersController))
#5 /Users/raj/Sites/webspace/sso/lib/Cake/Utility/ObjectCollection.php(130): call_user_func_array(Array, Array)
#6 [internal function]: ObjectCollection->trigger(Object(CakeEvent))
#7 /Users/raj/Sites/webspace/sso/lib/Cake/Event/CakeEventManager.php(246): call_user_func(Array, Object(CakeEvent))
#8 /Users/raj/Sites/webspace/sso/lib/Cake/Controller/Controller.php(671): CakeEventManager->dispatch(Object(CakeEvent))
#9 /Users/raj/Sites/webspace/sso/lib/Cake/Routing/Dispatcher.php(183): Controller->startupProcess()
#10 /Users/raj/Sites/webspace/sso/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#11 /Users/raj/Sites/webspace/sso/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2012-10-23 20:08:06 Error: [CAS_AuthenticationException] 
#0 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS/Client.php(1221): CAS_Client->validateCAS20('https://login.u...', '', NULL)
#1 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS/Client.php(1080): CAS_Client->isAuthenticated()
#2 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS.php(1100): CAS_Client->forceAuthentication()
#3 /Users/raj/Sites/webspace/sso/app/Controller/Component/CasAuthComponent.php(40): phpCAS::forceAuthentication()
#4 [internal function]: CasAuthComponent->startup(Object(UsersController))
#5 /Users/raj/Sites/webspace/sso/lib/Cake/Utility/ObjectCollection.php(130): call_user_func_array(Array, Array)
#6 [internal function]: ObjectCollection->trigger(Object(CakeEvent))
#7 /Users/raj/Sites/webspace/sso/lib/Cake/Event/CakeEventManager.php(246): call_user_func(Array, Object(CakeEvent))
#8 /Users/raj/Sites/webspace/sso/lib/Cake/Controller/Controller.php(671): CakeEventManager->dispatch(Object(CakeEvent))
#9 /Users/raj/Sites/webspace/sso/lib/Cake/Routing/Dispatcher.php(183): Controller->startupProcess()
#10 /Users/raj/Sites/webspace/sso/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#11 /Users/raj/Sites/webspace/sso/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2012-10-23 20:08:24 Error: [CAS_AuthenticationException] 
#0 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS/Client.php(1221): CAS_Client->validateCAS20('https://login.u...', '', NULL)
#1 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS/Client.php(1080): CAS_Client->isAuthenticated()
#2 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS.php(1100): CAS_Client->forceAuthentication()
#3 /Users/raj/Sites/webspace/sso/app/Controller/Component/CasAuthComponent.php(40): phpCAS::forceAuthentication()
#4 [internal function]: CasAuthComponent->startup(Object(UsersController))
#5 /Users/raj/Sites/webspace/sso/lib/Cake/Utility/ObjectCollection.php(130): call_user_func_array(Array, Array)
#6 [internal function]: ObjectCollection->trigger(Object(CakeEvent))
#7 /Users/raj/Sites/webspace/sso/lib/Cake/Event/CakeEventManager.php(246): call_user_func(Array, Object(CakeEvent))
#8 /Users/raj/Sites/webspace/sso/lib/Cake/Controller/Controller.php(671): CakeEventManager->dispatch(Object(CakeEvent))
#9 /Users/raj/Sites/webspace/sso/lib/Cake/Routing/Dispatcher.php(183): Controller->startupProcess()
#10 /Users/raj/Sites/webspace/sso/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#11 /Users/raj/Sites/webspace/sso/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2012-10-23 20:12:18 Error: [CAS_AuthenticationException] 
#0 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS/Client.php(1221): CAS_Client->validateCAS20('https://login.u...', '', NULL)
#1 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS/Client.php(1080): CAS_Client->isAuthenticated()
#2 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS.php(1100): CAS_Client->forceAuthentication()
#3 /Users/raj/Sites/webspace/sso/app/Controller/Component/CasAuthComponent.php(40): phpCAS::forceAuthentication()
#4 [internal function]: CasAuthComponent->startup(Object(UsersController))
#5 /Users/raj/Sites/webspace/sso/lib/Cake/Utility/ObjectCollection.php(130): call_user_func_array(Array, Array)
#6 [internal function]: ObjectCollection->trigger(Object(CakeEvent))
#7 /Users/raj/Sites/webspace/sso/lib/Cake/Event/CakeEventManager.php(246): call_user_func(Array, Object(CakeEvent))
#8 /Users/raj/Sites/webspace/sso/lib/Cake/Controller/Controller.php(671): CakeEventManager->dispatch(Object(CakeEvent))
#9 /Users/raj/Sites/webspace/sso/lib/Cake/Routing/Dispatcher.php(183): Controller->startupProcess()
#10 /Users/raj/Sites/webspace/sso/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#11 /Users/raj/Sites/webspace/sso/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2012-10-23 20:48:10 Error: [CAS_AuthenticationException] 
#0 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS/Client.php(1221): CAS_Client->validateCAS20('https://login.u...', '', NULL)
#1 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS/Client.php(1080): CAS_Client->isAuthenticated()
#2 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS.php(1100): CAS_Client->forceAuthentication()
#3 /Users/raj/Sites/webspace/sso/app/Controller/Component/CasAuthComponent.php(40): phpCAS::forceAuthentication()
#4 [internal function]: CasAuthComponent->startup(Object(UsersController))
#5 /Users/raj/Sites/webspace/sso/lib/Cake/Utility/ObjectCollection.php(130): call_user_func_array(Array, Array)
#6 [internal function]: ObjectCollection->trigger(Object(CakeEvent))
#7 /Users/raj/Sites/webspace/sso/lib/Cake/Event/CakeEventManager.php(246): call_user_func(Array, Object(CakeEvent))
#8 /Users/raj/Sites/webspace/sso/lib/Cake/Controller/Controller.php(671): CakeEventManager->dispatch(Object(CakeEvent))
#9 /Users/raj/Sites/webspace/sso/lib/Cake/Routing/Dispatcher.php(183): Controller->startupProcess()
#10 /Users/raj/Sites/webspace/sso/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#11 /Users/raj/Sites/webspace/sso/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2012-10-23 20:52:30 Error: [CAS_AuthenticationException] 
#0 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS/Client.php(1221): CAS_Client->validateCAS20('https://login.u...', '', NULL)
#1 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS/Client.php(1080): CAS_Client->isAuthenticated()
#2 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS.php(1100): CAS_Client->forceAuthentication()
#3 /Users/raj/Sites/webspace/sso/app/Controller/Component/CasAuthComponent.php(40): phpCAS::forceAuthentication()
#4 [internal function]: CasAuthComponent->startup(Object(UsersController))
#5 /Users/raj/Sites/webspace/sso/lib/Cake/Utility/ObjectCollection.php(130): call_user_func_array(Array, Array)
#6 [internal function]: ObjectCollection->trigger(Object(CakeEvent))
#7 /Users/raj/Sites/webspace/sso/lib/Cake/Event/CakeEventManager.php(246): call_user_func(Array, Object(CakeEvent))
#8 /Users/raj/Sites/webspace/sso/lib/Cake/Controller/Controller.php(671): CakeEventManager->dispatch(Object(CakeEvent))
#9 /Users/raj/Sites/webspace/sso/lib/Cake/Routing/Dispatcher.php(183): Controller->startupProcess()
#10 /Users/raj/Sites/webspace/sso/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#11 /Users/raj/Sites/webspace/sso/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2012-10-25 21:58:36 Error: [CAS_AuthenticationException] 
#0 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS/Client.php(1221): CAS_Client->validateCAS20('https://login.u...', '', NULL)
#1 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS/Client.php(1080): CAS_Client->isAuthenticated()
#2 /Users/raj/Sites/webspace/sso/app/Vendor/CAS/CAS/CAS.php(1100): CAS_Client->forceAuthentication()
#3 /Users/raj/Sites/webspace/sso/app/Controller/Component/CasAuthComponent.php(40): phpCAS::forceAuthentication()
#4 [internal function]: CasAuthComponent->startup(Object(UsersController))
#5 /Users/raj/Sites/webspace/sso/lib/Cake/Utility/ObjectCollection.php(130): call_user_func_array(Array, Array)
#6 [internal function]: ObjectCollection->trigger(Object(CakeEvent))
#7 /Users/raj/Sites/webspace/sso/lib/Cake/Event/CakeEventManager.php(246): call_user_func(Array, Object(CakeEvent))
#8 /Users/raj/Sites/webspace/sso/lib/Cake/Controller/Controller.php(671): CakeEventManager->dispatch(Object(CakeEvent))
#9 /Users/raj/Sites/webspace/sso/lib/Cake/Routing/Dispatcher.php(183): Controller->startupProcess()
#10 /Users/raj/Sites/webspace/sso/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest), Object(CakeResponse))
#11 /Users/raj/Sites/webspace/sso/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2012-10-25 21:58:55 Error: Fatal Error (1): Call to a member function allow() on a non-object in [/Users/raj/Sites/webspace/sso/app/Controller/AppController.php, line 201]
2012-10-25 21:58:55 Error: [FatalErrorException] Call to a member function allow() on a non-object
#0 /Users/raj/Sites/webspace/sso/lib/Cake/Error/ErrorHandler.php(162): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/Users/raj/Site...', 201)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/Users/raj/Site...', 201, Array)
#2 /Users/raj/Sites/webspace/sso/lib/Cake/Core/App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/Users/raj/Site...', 201, Array)
#3 /Users/raj/Sites/webspace/sso/lib/Cake/Core/App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2012-10-25 22:35:31 Error: [MissingViewException] View file "/Users/raj/Sites/webspace/sso/app/View/Worksheet/admin_submit_worksheet_form.ctp" is missing.
#0 /Users/raj/Sites/webspace/sso/lib/Cake/View/View.php(472): View->_getViewFileName(NULL)
#1 /Users/raj/Sites/webspace/sso/lib/Cake/Controller/Controller.php(957): View->render(NULL, NULL)
#2 /Users/raj/Sites/webspace/sso/lib/Cake/Routing/Dispatcher.php(193): Controller->render()
#3 /Users/raj/Sites/webspace/sso/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(WorksheetsController), Object(CakeRequest), Object(CakeResponse))
#4 /Users/raj/Sites/webspace/sso/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
